fairlearn/metrics/_metric_frame.py

```diff
@@ -189,6 +189,30 @@ class MetricFrame:
 
         .. deprecated:: 0.7.0
             `metric` will be removed in version 0.10.0, use `metrics` instead.
+
+    Examples
+    --------
+    >>> from fairlearn.metrics import MetricFrame, selection_rate
+    >>> y_true = [0,1,1,1,1,0,0,0,1,1]
```

Comment:
I think most of our library focusses on evaluating an ML model rather than the data, wouldn't it make more sense to use `y_pred` here instead of `y_true`?

@romanlutz @Thomasq99

Reply:
Should I then use a different `y_true` and `y_pred` for this case? In principle the `y_true` does not matter for the selection rate.  However, it feels odd to set `y_true=y_pred`.

Reply:
My preferred solution would be to use `y_true=None` and `y_pred=y_pred`. This will also hopefully prevent wrong usage of `selection_rate` by people who copy paste examples! :)

Reply:
It seems to me that this is not possible. Both `y_true` and `y_pred` need to be of the same shape. Therefore I propose the following solution: `y_true = [None]*10` and `y_pred=y_pred`. For now I have implemented it this way.

Reply:
That is a reasonable concern... Maybe we should use a different metric to avoid confusion? Let's say `accuracy_score`? Then we can provide `y_true` and `y_pred`. Or both `accuracy_score` AND `selection_rate` to show how to pass multiple?

Reply:
Oh I like the suggestion to add both!

Reply:
I did that in my new commit but I get the following test fail:
![image](https://user-images.githubusercontent.com/38749496/133287085-01d06d29-c6f5-4f18-a325-69e35b9a234e.png)

I can't figure out what is wrong. I copy the output from my IDE to the example directly and it seems to be exactly the same but still something is wrong. Can someone help me out?

Reply:
In our other docs, we have a `WHITESPACE` directive, which might help.

Reply:
I am not sure what that means, is this something I need to download on my IDE?

Reply:
I think that is specifically for writing documentation. I am writing code in the docstring so I am not sure whether that will work in code. Let me know if it does though!

Reply:
I misunderstood, but with some help I got it, seems to be working now!